kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,3 +1,7 @@
+local check_status = function(status)
+  return status > 99 and status <= 999
```

Comment:
This caused a failure due to not nil checking. Fixed locally and will push once specs have run.